pkg/advancedapi/advancedApi.go

```diff
@@ -130,10 +139,14 @@ func SetDocumentController(resolverClient register.ResolverClient, identity regi
 }
 
 // SetDocumentCreator Set register document creator.
-func SetDocumentCreator(resolverClient register.ResolverClient, identity register.RegisteredIdentity, creator *register.Issuer) error {
-	originalDoc, err := GetRegisterDocument(resolverClient, identity.Did())
-	if err != nil {
-		return err
+func SetDocumentCreator(resolverClient register.ResolverClient, originalDoc *register.RegisterDocument, identity register.RegisteredIdentity, creator *register.Issuer) error {
```

Comment:
Creator vs Controller vs Control-delegation - are there some words (in docs  / elsewhere) about what these mean?
Or is SetController & Control-delegation one and the same thing? Looking at `Test_can_validate_allowed_for_control_with_controller_doc` - it would appear yes - or do I misunderstand?

Reply:
Ok, after you explained - I understand what they mean.

So maybe part of the extra doc push - as you suggest in https://github.com/Iotic-Labs/iotics-identity-go/pull/20#discussion_r717685671